File: lc744.java
public class lc744 {
    // 744. Find Smallest Letter Greater Than Target
    public static void main(String[] args) {
        char letters[]={'c','f','j'};
        char target='a';
        char ans=' ';
        for (int i=0;i< letters.length;i++){
            if (letters[i]>target){
                ans=letters[i];
                break;
            }
        }
        if (ans==' '){
            ans=letters[0];
        }
        System.out.println(ans);
    }
}

File: lc744.py
from typing import List


class Solution:
    def nextGreatestLetter(self, letters: List[str], target: str) -> str:
        a=letters[0]
        for i in letters:
            if i>target:
                a=i
                break
        return a
print(Solution.nextGreatestLetter(0,["c","f","j"],"a"))